packages/dashboard-frontend/src/store/ServerConfig/helpers.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2018-2024 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+
+export function isSourceAllowed(allowedSourceUrls: string[], url: string): boolean {
+  for (const allowedSourceUrl of allowedSourceUrls) {
+    if (allowedSourceUrl.indexOf('*') !== -1) {
```

Comment:
```suggestion
    if (allowedSourceUrl.includes('*')) {
```

Reply:
@olexii4 I believe it's a matter of taste